script.myepisodes/resources/__init__.py

```diff
@@ -0,0 +1 @@
+# Dummy file to make this directory a package.
```

Comment:
shoudn't be needed anymore, however I understand if this is necessary because of leia and below.

Reply:
I've removed it, this is for Matrix and up anyway.